src/persistence.interfaces.ts

```diff
@@ -98,7 +98,7 @@ export interface IPersistence {
         expires: string,
         domain: string,
         maxCookieSize: number
-    ): void;
+    ): string | undefined;
```

Comment:
i feel like `| undefined` may not be needed here and is implicit with the compiler